inc/saiswitch.h

```diff
@@ -3096,6 +3096,46 @@ typedef sai_status_t (*sai_switch_mdio_write_fn)(
         _In_ uint32_t number_of_registers,
         _In_ const uint32_t *reg_val);
 
+/**
+ * @brief Switch MDIO clause 22 read API
+ *
+ * Provides clause 22 read access API for devices connected to MDIO from NPU SAI.
+ *
+ * @objects switch_id SAI_OBJECT_TYPE_SWITCH
+ *
+ * @param[in] switch_id Switch Id
+ * @param[in] device_addr Device address(PHY/lane/port MDIO address)
+ * @param[in] start_reg_addr Starting register address to read
+ * @param[in] number_of_registers Number of consecutive registers to read
+ * @param[out] reg_val Register read values
+ */
+typedef sai_status_t (*sai_switch_mdio_cl22_read_fn)(
```

Comment:
Type `sai_switch_mdio_cl22_read_fn` is same as `sai_switch_mdio_read_fn`. It looks not needed to define it.

Reply:
In the first commit: https://github.com/opencomputeproject/SAI/commit/912c4c4e5c73808e06f535677e82e99b46500117 The new cl22 functions had the type you suggested. But it would not pass the parse.pl sanity check. I changed the cl22 function to the new type in the last commit: https://github.com/opencomputeproject/SAI/commit/0fb1c06d137eff057f61c8ef9fef0ea234514da5